apple/OmnivoreKit/Sources/Models/DataModels/FeedItem.swift

```diff
@@ -58,6 +58,19 @@ public extension LinkedItem {
     return URL(string: pageURLString ?? "")
   }
 
+  var labelsJSONString: String {
```

Comment:
It would be nicer to do this in the `LinkedItemLabel` extension but thats in the models.